docs/source/llms/gateway/index.rst

```diff
@@ -899,6 +896,85 @@ The results of the query are:
          }
        }
 
+Streaming
+~~~~~~~~~
+
+Some providers support streaming responses. Streaming responses are useful when you want to
```

Comment:
Can we say "The following providers supports ..." instead of `Some providers ...`?

Reply:
Never mind this comment.